File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/drgs/DRG0201U00GrdOrderInfo.java
package nta.med.data.model.ihis.drgs;

import java.sql.Timestamp;

public class DRG0201U00GrdOrderInfo {
	private Double drgBunho;
	private String bunho;
	private Timestamp orderDate;
	private Timestamp jubsuDate;
	private Timestamp drgJubsuDate;
	private String jubsuTime;
	private String doctor;
	private String doctorName;
	private String gwa;
	private String buseoName;
	private Timestamp actDate;
	private Timestamp actTime;
	private String actYn;
	private Timestamp sunabDate;
	private Timestamp chulgoDate;
	private String actUserName;
	private String wonyoiOrderYn;
	private String disGubun;
	private String orderRemark;
	private String drgRemark;
	private Double fkout1001;
	private String ifDataSendYn;
	public DRG0201U00GrdOrderInfo(Double drgBunho, String bunho, Timestamp orderDate, Timestamp jubsuDate,
			Timestamp drgJubsuDate, String jubsuTime, String doctor, String doctorName, String gwa, String buseoName,
			Timestamp actDate, Timestamp actTime, String actYn, Timestamp sunabDate, Timestamp chulgoDate,
			String actUserName, String wonyoiOrderYn, String disGubun, String orderRemark, String drgRemark,
			Double fkout1001, String ifDataSendYn) {
		super();
		this.drgBunho = drgBunho;
		this.bunho = bunho;
		this.orderDate = orderDate;
		this.jubsuDate = jubsuDate;
		this.drgJubsuDate = drgJubsuDate;
		this.jubsuTime = jubsuTime;
		this.doctor = doctor;
		this.doctorName = doctorName;
		this.gwa = gwa;
		this.buseoName = buseoName;
		this.actDate = actDate;
		this.actTime = actTime;
		this.actYn = actYn;
		this.sunabDate = sunabDate;
		this.chulgoDate = chulgoDate;
		this.actUserName = actUserName;
		this.wonyoiOrderYn = wonyoiOrderYn;
		this.disGubun = disGubun;
		this.orderRemark = orderRemark;
		this.drgRemark = drgRemark;
		this.fkout1001 = fkout1001;
		this.ifDataSendYn = ifDataSendYn;
	}
	public Double getDrgBunho() {
		return drgBunho;
	}
	public void setDrgBunho(Double drgBunho) {
		this.drgBunho = drgBunho;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public Timestamp getOrderDate() {
		return orderDate;
	}
	public void setOrderDate(Timestamp orderDate) {
		this.orderDate = orderDate;
	}
	public Timestamp getJubsuDate() {
		return jubsuDate;
	}
	public void setJubsuDate(Timestamp jubsuDate) {
		this.jubsuDate = jubsuDate;
	}
	public Timestamp getDrgJubsuDate() {
		return drgJubsuDate;
	}
	public void setDrgJubsuDate(Timestamp drgJubsuDate) {
		this.drgJubsuDate = drgJubsuDate;
	}
	public String getJubsuTime() {
		return jubsuTime;
	}
	public void setJubsuTime(String jubsuTime) {
		this.jubsuTime = jubsuTime;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getBuseoName() {
		return buseoName;
	}
	public void setBuseoName(String buseoName) {
		this.buseoName = buseoName;
	}
	public Timestamp getActDate() {
		return actDate;
	}
	public void setActDate(Timestamp actDate) {
		this.actDate = actDate;
	}
	public Timestamp getActTime() {
		return actTime;
	}
	public void setActTime(Timestamp actTime) {
		this.actTime = actTime;
	}
	public String getActYn() {
		return actYn;
	}
	public void setActYn(String actYn) {
		this.actYn = actYn;
	}
	public Timestamp getSunabDate() {
		return sunabDate;
	}
	public void setSunabDate(Timestamp sunabDate) {
		this.sunabDate = sunabDate;
	}
	public Timestamp getChulgoDate() {
		return chulgoDate;
	}
	public void setChulgoDate(Timestamp chulgoDate) {
		this.chulgoDate = chulgoDate;
	}
	public String getActUserName() {
		return actUserName;
	}
	public void setActUserName(String actUserName) {
		this.actUserName = actUserName;
	}
	public String getWonyoiOrderYn() {
		return wonyoiOrderYn;
	}
	public void setWonyoiOrderYn(String wonyoiOrderYn) {
		this.wonyoiOrderYn = wonyoiOrderYn;
	}
	public String getDisGubun() {
		return disGubun;
	}
	public void setDisGubun(String disGubun) {
		this.disGubun = disGubun;
	}
	public String getOrderRemark() {
		return orderRemark;
	}
	public void setOrderRemark(String orderRemark) {
		this.orderRemark = orderRemark;
	}
	public String getDrgRemark() {
		return drgRemark;
	}
	public void setDrgRemark(String drgRemark) {
		this.drgRemark = drgRemark;
	}
	public Double getFkout1001() {
		return fkout1001;
	}
	public void setFkout1001(Double fkout1001) {
		this.fkout1001 = fkout1001;
	}
	public String getIfDataSendYn() {
		return ifDataSendYn;
	}
	public void setIfDataSendYn(String ifDataSendYn) {
		this.ifDataSendYn = ifDataSendYn;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Drgs/DRG0201U00GrdOrderInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Drgs
{
    [Serializable]
    public class DRG0201U00GrdOrderInfo
    {
        private String _drgBunho;
        private String _bunho;
        private String _orderDate;
        private String _jubsuDate;
        private String _drgJubsuDate;
        private String _jubsuTime;
        private String _doctor;
        private String _doctorName;
        private String _gwa;
        private String _buseoName;
        private String _actDate;
        private String _actTime;
        private String _actYn;
        private String _sunabDate;
        private String _chulgoDate;
        private String _actUserName;
        private String _wonyoiOrderYn;
        private String _disGubun;
        private String _orderRemark;
        private String _drgRemark;
        private String _fkout1001;
        private String _ifDataSendYn;

        public String DrgBunho
        {
            get { return this._drgBunho; }
            set { this._drgBunho = value; }
        }

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String OrderDate
        {
            get { return this._orderDate; }
            set { this._orderDate = value; }
        }

        public String JubsuDate
        {
            get { return this._jubsuDate; }
            set { this._jubsuDate = value; }
        }

        public String DrgJubsuDate
        {
            get { return this._drgJubsuDate; }
            set { this._drgJubsuDate = value; }
        }

        public String JubsuTime
        {
            get { return this._jubsuTime; }
            set { this._jubsuTime = value; }
        }

        public String Doctor
        {
            get { return this._doctor; }
            set { this._doctor = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String BuseoName
        {
            get { return this._buseoName; }
            set { this._buseoName = value; }
        }

        public String ActDate
        {
            get { return this._actDate; }
            set { this._actDate = value; }
        }

        public String ActTime
        {
            get { return this._actTime; }
            set { this._actTime = value; }
        }

        public String ActYn
        {
            get { return this._actYn; }
            set { this._actYn = value; }
        }

        public String SunabDate
        {
            get { return this._sunabDate; }
            set { this._sunabDate = value; }
        }

        public String ChulgoDate
        {
            get { return this._chulgoDate; }
            set { this._chulgoDate = value; }
        }

        public String ActUserName
        {
            get { return this._actUserName; }
            set { this._actUserName = value; }
        }

        public String WonyoiOrderYn
        {
            get { return this._wonyoiOrderYn; }
            set { this._wonyoiOrderYn = value; }
        }

        public String DisGubun
        {
            get { return this._disGubun; }
            set { this._disGubun = value; }
        }

        public String OrderRemark
        {
            get { return this._orderRemark; }
            set { this._orderRemark = value; }
        }

        public String DrgRemark
        {
            get { return this._drgRemark; }
            set { this._drgRemark = value; }
        }

        public String Fkout1001
        {
            get { return this._fkout1001; }
            set { this._fkout1001 = value; }
        }

        public String IfDataSendYn
        {
            get { return this._ifDataSendYn; }
            set { this._ifDataSendYn = value; }
        }

        public DRG0201U00GrdOrderInfo() { }

        public DRG0201U00GrdOrderInfo(String drgBunho, String bunho, String orderDate, String jubsuDate, String drgJubsuDate, String jubsuTime, String doctor, String doctorName, String gwa, String buseoName, String actDate, String actTime, String actYn, String sunabDate, String chulgoDate, String actUserName, String wonyoiOrderYn, String disGubun, String orderRemark, String drgRemark, String fkout1001, String ifDataSendYn)
        {
            this._drgBunho = drgBunho;
            this._bunho = bunho;
            this._orderDate = orderDate;
            this._jubsuDate = jubsuDate;
            this._drgJubsuDate = drgJubsuDate;
            this._jubsuTime = jubsuTime;
            this._doctor = doctor;
            this._doctorName = doctorName;
            this._gwa = gwa;
            this._buseoName = buseoName;
            this._actDate = actDate;
            this._actTime = actTime;
            this._actYn = actYn;
            this._sunabDate = sunabDate;
            this._chulgoDate = chulgoDate;
            this._actUserName = actUserName;
            this._wonyoiOrderYn = wonyoiOrderYn;
            this._disGubun = disGubun;
            this._orderRemark = orderRemark;
            this._drgRemark = drgRemark;
            this._fkout1001 = fkout1001;
            this._ifDataSendYn = ifDataSendYn;
        }

    }
}